lnrpc/routerrpc/router.proto

```diff
@@ -209,6 +209,43 @@ message ChannelUpdate {
 	// satoshi.
 	uint32 fee_rate = 9;
 }
+message ResetMissionControlRequest{}
+
+message ResetMissionControlResponse{}
+
+message QueryMissionControlRequest {}
+
+/// QueryMissionControlResponse contains mission control state per node.
+message QueryMissionControlResponse {
+    repeated NodeHistory nodes = 1;
+}
+
+/// NodeHistory contains the mission control state for a particular node.
+message NodeHistory {
+    /// Node pubkey
+    bytes pubkey = 1;
+
+    /// Time stamp of last failure. Set to zero if no failure happened yet.
+    int64 last_fail = 2;
+
+    /// Estimation of success probability for channels not in the channel array.
```

Comment:
The array only contains channels for which we have specific information. For channels that we never tried, we can only look at node level failures. Those are assumed to have happened regardless of the outgoing channel.

Reply:
gotcha, i'm wondering then if naming it `node_success_prob` is more clear. up to you

Reply:
Yes, the thing is that it is not a node probability. It is an estimate for channels that we don't have any other info on. In the future, this could also be (for example) an average of all channels that we do have info on. I found the concept of node probability and how to blend that with channel probabilities not very clear.